Presentation.Web/app/components/user-notification/user-notification-modal.controller.ts

```diff
@@ -0,0 +1,242 @@
+﻿module Kitos.UserNotifications {
+    "use strict";
+
+    export class UserNotificationsModalController {
+
+        mainGridOptions;
+        mainGrid; // We don't set the grid. It is being set by the options that we define. But we need access to this in order to manually force a refresh of the data.
+
+        static $inject: Array<string> = [
+            "$scope",
+            "notify",
+            "$uibModalInstance",
+            "userNotificationService",
+            "contextType"
+        ];
+
+        constructor(
+            private readonly $scope,
+            private readonly notify,
+            private readonly $uibModalInstance: ng.ui.bootstrap.IModalServiceInstance,
+            private readonly userNotificationService: Services.UserNotification.IUserNotificationService,
+            private readonly contextType: Models.UserNotification.ObjectType
+        ) {
+            this.mainGridOptions = {
+                dataSource: {
+                    transport: {
+                        read: {
+                            url: `/api/v1/user-notification/organization/${this.$scope.user.currentOrganizationId}/context/${this.contextType}/user/${this.$scope.user.id}`,
```

Comment:
Resolve user via user service som dependency til denne i stedet for at antage at den ligger på scope